src/app/profile/profile.model.ts

```diff
@@ -0,0 +1,13 @@
+export interface Profile {
+    firstName: string;
+    lastName: string;
+    email: string;
+    phoneNumbers: [
+        {
+            number: string;
+        },
+        {
+            number: string;
+        }
+    ]
+}
```

Comment:
QUESTION: I wanted to have this be the type for my profile property to follow. It didnt work after I made the data come in via a service. To see how it was before reference[ ](https://github.com/lmahoney1218/angular-tour-of-heroes/pull/1/commits/ad5d5177a0d71913e3a644cea24013064c2e6852)

Reply:
Ah i didn't see this before my other comments.  You need to define a type for Phone numbers.

```
export interface Profile {
  firstName: string,
  lastName: string,
  email: string,
  phoneNumbers: PhoneNumber[]
}

export interface PhoneNumber {
  number: string 
}
```

Reply:
Awesome! It kept erroring at the array when I had it all as one interface. It didn't occur to me to implement it like this, but it makes total sense now. Thank you!